app/examples/vivaldi

```diff
@@ -0,0 +1,83 @@
+" Options
+set adblocker=update
+set cache=full
+set devtoolsposition=vsplit
+set favicons=30day
+set mintabwidth=0
+set permissioncamera=ask
+set permissiondisplaycapture=ask
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set search=https://google.com/search?q=%s
+set splitright
+set suspendonrestore=none
+set tabreopenposition=previous
+set windowtitle=title
+
+" Search Engines
+set search=https://www.bing.com/search?q=%s
+
+" Mappings
+nmap <Up> <scrollUp>
+nmap <Down> <scrollDown>
+nmap <Left> <scrollLeft>
+nmap <Right> <scrollRight>
+map <PageUp> <scrollPageUp>
+map <PageDown> <scrollPageDown>
+map <Home> <scrollTop>
+map <End> <scrollBottom>
+map <F3> <toSearchMode>
+nmap <F2> <toCommandMode>
```

Comment:
Can we map these keys in normal mode only? `PageDown` etc. have very different functions when typing text for example.

Also, are the `<F2>` and `<F3>` keys toggles for switching back and forth between these modes? I that case it would be nice to also add these mappings to search and command mode to go back to normal mode.

Reply:
`<F2>` is a toggle. `<F3>` is not. I've changed this in the latest commits.

Reply:
I need to change the behavior of `<F3>`- and `<C-g>`-related commands to better match Vivaldi. I will commit those changes once they work.

Reply:
Great, let me know when I can check it again.

Reply:
I've fixed this in the latest commit.